src/feed-list.scss

```diff
@@ -0,0 +1,144 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+
+.#{$block} {
+  $fd-feed-list-padding-start: 1rem;
+  $fd-feed-list-padding-end: 0.5rem;
+  $fd-feed-list-margin-start: 0.75rem;
+  $fd-feed-list-margin-end: 0;
+  $fd-feed-list-padding-group: 0.75rem;
+  $fd-feed-list-icon-color: var(--sapContent_IconColor);
+
+  @include fd-reset();
+  @include fd-flex();
+
+  background-color: var(--sapField_Background);
+  max-width: 40rem;
+
+  .#{$block}__thumb {
+    @include fd-reset();
+
+    display: inline-flex;
+    margin-left: $fd-feed-list-margin-end;
+    margin-right: $fd-feed-list-margin-start;
+
+    @include fd-rtl() {
+      margin-left: $fd-feed-list-margin-start;
+      margin-right: $fd-feed-list-margin-end;
+    }
+  }
+
+  &__body {
+    @include fd-reset();
+
+    display: flex;
+    padding: 1rem;
+
+    & + .#{$block}__body {
+      border-top: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
+    }
+  }
+
+  &--no-border {
+    .#{$block}__body {
+      padding: 1rem 1rem 0.75rem;
+      border-top: none;
+
+      & + .#{$block}__body {
+        padding-top: $fd-feed-list-padding-group;
+        padding-bottom: $fd-feed-list-padding-group;
+      }
+
+      &:last-child {
+        padding-bottom: 1rem;
+      }
+    }
+  }
+
+  &__link {
+    &--more {
+      text-transform: uppercase;
+    }
+  }
+
+  &__actions {
+    @include fd-reset();
+
+    float: right;
+    margin: 0.5rem;
+    position: relative;
+    top: -1rem;
+    right: -1rem;
+
+    @include fd-rtl() {
+      float: left;
+      right: 1rem;
+      left: -1rem;
+    }
+  }
+
+  &--group {
+    flex-direction: column;
+  }
+
+  &--link {
+    @include fd-fiori-focus();
+
+    cursor: pointer;
+  }
+
+  &__name {
+    @include fd-reset();
+
+    font-weight: bold;
+  }
+
+  &__text {
+    @include fd_reset();
+  }
+
+  &__footer {
+    @include fd-reset();
+
+    display: flex;
+    padding-top: 0.375rem;
+
+    &--byline {
+      color: var(--sapNeutralTextColor);
+
+      &:not(:last-child)::after {
+        content: '\2022';
+        padding: 0 0.25rem;
+      }
+    }
+  }
+
+  &--s {
+    .#{$block}__wrapper {
+      @include fd-reset();
+
+      display: flex;
+      justify-content: space-between;
+
+      &--header {
+        display: flex;
+      }
+    }
+
+    .#{$block}__body {
+      display: block;
+    }
+
+    .#{$block}__text {
+      padding-top: 0.625rem;
+    }
+
+    .#{$block}__action {
+      @include fd-reset();
+
+      padding-left: 0.75rem;
+      padding-bottom: 0.25rem;
+    }

```

Comment:
Could you add rtl support there? (In this case you would neeed to change `0.75rem` to variable

Reply:
Added